yarn.lock

```diff
@@ -12567,6 +12567,22 @@ instantsearch.css@7.3.1:
   resolved "https://registry.yarnpkg.com/instantsearch.css/-/instantsearch.css-7.3.1.tgz#7ab74a8f355091ae040947a9cf5438f379026622"
   integrity sha512-/kaMDna5D+Q9mImNBHEhb9HgHATDOFKYii7N1Iwvrj+lmD9gBJLqVhUw67gftq2O0QI330pFza+CRscIwB1wQQ==
 
+instantsearch.js@4.33.2:
```

Comment:
the dependency in React InstantSearch hooks should be `^` so this doesn't get duplicated

Reply:
I deduped the lock file in d6bac79b7761c70b98670941fcb0f6add9b6ba82.